packages/ui/src/theme/tokens/components/switchField.ts

```diff
@@ -11,67 +12,73 @@ import {
   TransitionDurationValue,
 } from '../types/designToken';
 
-interface SwitchFieldDisabledTokens {
-  opacity: DesignToken<OpacityValue>;
-}
-
-interface SwitchFieldFocusedTokens {
-  shadow: DesignToken<ShadowValue>;
-}
-
-interface SwitchFieldSizeTokens {
-  fontSize: DesignToken<FontSizeValue>;
-}
-
-interface SwitchFieldLabelTokens {
-  padding: DesignToken<SpaceValue>;
-}
-
-interface SwitchFieldThumbTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
-  borderColor: DesignToken<BorderColorValue>;
-  borderRadius: DesignToken<BorderRadiusValue>;
-  checked: SwitchFieldThumbCheckedTokens;
-  transition: SwitchFieldThumbTransitionTokens;
-  width: DesignToken<SpaceValue>;
-}
-
-interface SwitchFieldThumbTransitionTokens {
-  duration: DesignToken<TransitionDurationValue>;
-}
-
-interface SwitchFieldThumbCheckedTokens {
-  transform: DesignToken<TransformValue>;
-}
-interface SwitchFieldTrackTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
-  borderRadius: DesignToken<BorderRadiusValue>;
-  checked: SwitchFieldTrackCheckedTokens;
-  height: DesignToken<SpaceValue>;
-  padding: DesignToken<SpaceValue>;
-  transition: SwitchFieldTrackTransitionTokens;
-  width: DesignToken<SpaceValue>;
-  _error: SwitchFieldTrackCheckedTokens;
-}
-
-interface SwitchFieldTrackTransitionTokens {
-  duration: DesignToken<TransitionDurationValue>;
-}
-
-interface SwitchFieldTrackCheckedTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
-}
-
-export interface SwitchFieldTokens {
-  fontSize: DesignToken<FontSizeValue>;
-  _disabled: SwitchFieldDisabledTokens;
-  _focused: SwitchFieldFocusedTokens;
-  large: SwitchFieldSizeTokens;
-  small: SwitchFieldSizeTokens;
-  label: SwitchFieldLabelTokens;
-  thumb: SwitchFieldThumbTokens;
-  track: SwitchFieldTrackTokens;
-}
+type SwitchFieldDisabledTokens<OutputType> = DesignTokenProperties<
```

Comment:
By no means do i have a strong feeling (and my opinion is somewhat subjective) , but one of the things I've been trying to do is remove interfaces that are only used once unless they are more complex/contain nested interfaces. Meaning the values for `SwitchFieldDisabledTokens` and `SwitchFieldLabelToken` would just be directly referenced in `SwitchFieldTokens`, but would keep `SwitchFieldSizeTokens` (used multiple times), `SwitchFieldThumbTokens` and `SwitchFieldTrackTokens` (complex):
```
// no guarantee of syntactical correctness  😅
export type SwitchFieldTokens<OutputType = unknown> =
  DesignTokenProperties<'fontSize'> & {
    _disabled?: DesignTokenProperties<'opacity', OutputType>;
    _focused?: SwitchFieldFocusedTokens<OutputType>;
    large?: SwitchFieldSizeTokens<OutputType>;
    small?: SwitchFieldSizeTokens<OutputType>;
    label?: DesignTokenProperties<'padding', OutputType>;
    thumb?: SwitchFieldThumbTokens<OutputType>;
    track?: SwitchFieldTrackTokens<OutputType>;
  };